test/resources/IssuerFraudRecords.spec.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var stripe = require('../testUtils').getSpyableStripe();
+var expect = require('chai').expect;
+
+describe('IssuerFraudRecord Resource', function() {
+  describe('retrieve', function() {
+    it('Sends the correct request for issfr ID', function() {
```

Comment:
Can you truncate this down to just "Sends the correct request" for consistency?

Reply:
👍